templates/core.conf.erb

```diff
@@ -2,5 +2,8 @@
   "file": 1,
   "format": 1
 }{
-  "download_location": "<%= @download_location %>"
-}
\ No newline at end of file
+  "download_location": "<%= @download_location %>",
+  "move_completed_path": "<%= @move_completed_path %>",
```

Comment:
None of these behave nicely if they are undefined - you end up with

    "move_completed_path": "undef",

(or something -- not sure, but it's not handled here at least)